etl/collections/model.py

```diff
@@ -483,6 +492,29 @@ def v(self):
     def d(self):
         return self.dimensions
 
+    @property
+    def catalog_path(self) -> Optional[str]:
+        return self._catalog_path
+
+    @catalog_path.setter
+    def catalog_path(self, value: str) -> None:
+        assert "#" in value, "Catalog path should be in the format `path#name`."
+        self._catalog_path = value
+
+    @property
+    def local_config_path(self) -> Path:
+        # energy/latest/energy_prices#energy_prices -> export/multidim/energy/latest/energy_prices/config.yml
+        assert self.catalog_path
+        if self._collection_type is None:
+            raise ValueError("_collection_type must have a value!")
+        return EXPORT_DIR / self._collection_type / (self.catalog_path.replace("#", "/") + ".config.json")
```

Comment:
We save explorers into `explorers` folder, but `_collection_type` is `explorer`. It's a bit annoying that we use singular `multidim` but plural `explorers`. I wouldn't do it as part of this PR, but sometimes in the future we can rename `explorers` to `explorer` (I wouldn't rush it, it's just a cosmetic thing and there are several open PRs that'd need to be rebased).

Reply:
good catch; I'll adapt this bit